_posts/kubernetes/1200-09-01-K8-general-kubernetes-questions.md

```diff
@@ -35,9 +35,41 @@ It is possible to deploy an ingress resource in the same way you can deploy any
 ## Can V1 kube and V2 kube live on the same cluster?
 Yes. However, in the Spinnaker UI you will see every K8s resource twice. Once, when the V1 provider sees it, and once, when the V2 provider sees it.
 
+## is it possible to apply secrets using https://github.com/futuresimple/helm-secrets?
+Today that is not possible
 
+## How do we handle deployment in multiples cluster? For example, if our environment includes more than one k8s cluster
+With the account concept in K8s, an account is just a k8s cluster, when you configure an account you configure it to use an specific kube config file, this define how we communicate and authenticate on our cluster, if you have multiple cluster you can configure multiple accounts within spinnaker using different kube config files that have the address for the API, when you set the deployment step you setup a different account to deploy to and spinnaker will use the credentials found in the kube config.
```

Comment:
I prefer to use `Kubernetes` instead of `K8s` but if we use `K8s` we should make sure the case is consistent. There's an instance of `k8s` here.

Reply:
`kube config` should be `kubeconfig`.

Reply:
s/define/defines
s/cluster/clusters

Reply:
Changed all K8s into Kubernetes
Changed kube config into kubeconfig
changed the define into defines
changed cluster into clusters except for "an account is just a Kubernetes cluster" this looks good as singular right?